content/intermediate/templates/language-extensions/index.md

```diff
@@ -140,25 +177,47 @@ You'll now update the `language-extensions.yaml` template to add a CloudWatch da
 
 Save the template file, and proceed to the next steps.
 
 You'll now update your existing stack that you created in Part 1. To do so, follow steps shown next:
-
+:::::tabs{variant="container"}
+
+::::tab{id="cloud9" label="Cloud9"}
+1. Update the stack `cfn-workshop-language-extensions` by running the following AWS CLI command.
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws cloudformation update-stack \
+    --stack-name cfn-workshop-language-extensions \
+    --template-body file://language-extensions.yaml \
+    --capabilities CAPABILITY_AUTO_EXPAND
+1. CloudFormation returns the following output.
+:::code{language=json showLineNumbers=false showCopyAction=false}
+{
+    "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-language-extensions/466df9e0-0dff-08e3-8e2f-5088487c4896"
+}
+:::
+1. Wait until the `UPDATE` operation is complete, by using the [wait stack-update-complete](https://docs.aws.amazon.com/cli/latest/reference/cloudformation/wait/stack-update-complete.html) AWS CLI command.
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+aws cloudformation wait stack-update-complete \
+    --stack-name cfn-workshop-language-extensions
+:::
+::::
+::::tab{id="local" label="Local development"}
 1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/).
-2. From the left navigation panel, select the **Stacks** tab. Select the `language-extensions` stack you created earlier.
+2. From the left navigation panel, select the **Stacks** tab. Select the `cfn-workshop-language-extensions` stack you created earlier.
 3. From the top-right menu, choose **Update**.
 4. Under **Prerequisite - Prepare template,** select **Replace current template** and choose **Upload a template file**. Select **Choose file**, and supply the `language-extensions.yaml` template you updated earlier, and then choose **Next**.
 5. On **Specify Stack details** page, leave the configuration as it is. Choose **Next**.
 6. On **Configure Stack options**, leave the configuration as it is. Choose **Next**.
 7. On **Review** page, review the contents of the page. At the bottom of the page, choose to acknowledge all the capabilities shown in the **Capabilities and transforms** section.
 8. Choose **Submit**. Refresh the stack creation page until you see the stack in the `UPDATE_COMPLETE` status.
-9. Navigate to the [CloudWatch console](https://console.aws.amazon.com/cloudwatch/). From the right navigation panel, choose **Dashboards**. You should see the dashboard you just created.
-
+::::
+:::::
+Navigate to the [CloudWatch console](https://console.aws.amazon.com/cloudwatch/). From the right navigation panel, choose **Dashboards**. You should see the dashboard you just created.
```

Comment:
Should we add in some additional guidance around inspecting the dashboard source:
* open dashboard
* choose Actions
* choose View/edit source
You should see the JSON for the dashboard that matches your YAML from `language-extensions.yaml`